ata_pipeline0/main.py

```diff
@@ -24,14 +24,27 @@
 
 def handler(event, context):
     # Note: this is invoked by an event-driven, async method (s3 trigger) so the return value is discarded
-    print("request: {}".format(json.dumps(event)))
-    return {"hi": "there"}
+    bucket_name = event["Records"][0]["s3"]["bucket"]["name"]
+    site_name = re.findall("lnl-snowplow-(.*)", bucket_name)[0]
```

Comment:
❓ Is `site_name` guaranteed to be one of the four partners, or is it possible that it might be something we don't want, say, `texas-tribune` from the `lnl-snowplow-texas-tribune` bucket name? Anyhow, perhaps it's worth checking that `site_name in [*SiteName]`  before passing it into `run_pipeline` as a parameter?